docs/webpack-dev-server.config.js

```diff
@@ -51,14 +51,7 @@ const config = {
   module: {
     //eslint loader
     preLoaders: [
-      {
-        test: /\.(js|jsx)$/,
-        loader: 'eslint-loader',
-        include: [path.resolve(__dirname, '../src')],
-        exclude: [
-          path.resolve(__dirname, '../src/svg-icons'),
-        ],
-      },
+
```

Comment:
I wanted to try to make it optional, but then I see it doesn't work well with docs either, so: `npm start` !== `npm run lint`. Let's kill it before it lays eggs :scream_cat: